packages/sui-mono/README.md

```diff
@@ -61,7 +61,9 @@ sui-mono release
 The first one will provide information if anything requires a release.
 The second one will release the packages
 
-> Your packages must implement script `npm run build` that will be executed before any release.
+> Your packages must implement script `npm run build` or `npm run prepublish` that will be executed before any release.
+
+**sui-mono only creates releases for packages that contain `fix`, `perf` or `feat` commits. Otherwise, nothing will be done or outputted.**
```

Comment:
Regarding this, I was wondering if refactor should create a new release.

At the end code has changed. And maybe those refactors introduce less bundle size or a bit of execution times improvements or just more concise readings.

A lot of times i do refactors and label them as feature just to get a release